common/test/acceptance/tests/studio/test_studio_outline.py

```diff
@@ -198,71 +205,129 @@ def _build_fixture(self, unit_state):
             subsection if unit_state.publish_state == self.PublishState.NEVER_PUBLISHED
             else subsection.add_children(
                 XBlockFixtureDesc('vertical', name, metadata={
-                    'visible_to_staff_only': True if unit_state.is_locked else None
+                    'visible_to_staff_only': True if unit_state.is_locked else None,
+                    # Below 1 is the hardcoded group ID for Audit
+                    'group_access': {ENROLLMENT_TRACK_PARTITION_ID: [1]} if unit_state.is_restricted else {},
+                    'user_partitions': [
+                        create_user_partition_json(
+                            ENROLLMENT_TRACK_PARTITION_ID,
+                            'Name of Enrollment Track Partition',
+                            'Enrollment Track Partition',
+                            [
+                                Group(1, 'Audit')
+                            ],
+                            scheme="cohort"
```

Comment:
Scheme should not be cohort here (if we are trying to mimic enrollment tracks). You could use cohorts though if that is easier-- in that case, I just wouldn't make them "sound like" enrollment tracks.